### ISSUE TITLE
Stepper Motor Reversed

### ISSUE THREAD

**Issue by Alphacode** [gvdberg1966/tronxy-x5sa#1]
Extrusion Motor was reversed after install direct from 330 update folder

**Comment by moyjas1980** [comment]
yep seeing the same issue. I was able to overcome this issue by repinning the extruder plug in reverse order.
